packages/wallet/src/ui/Label.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ComponentProps, ReactNode } from 'react'
+
+type LabelProps = Omit<
```

Comment:
just a little consistency something here :D , if the previous Input.tsc had Omit details in one row, then this one could have it the same, or Input.tsx can have it in multiple lines as well. Whicever suits you best.

Reply:
This is how Prettier formats at the moment.